pallets/common/src/benchs/user_builder.rs

```diff
@@ -120,7 +120,7 @@ impl<T: Trait> UserBuilder<T> {
         self_update!(self, balance, b.into())
     }
 
-    fn make_key_pair(name: &'static str, u: u32) -> (T::AccountId, Option<SecretKey>) {
+    pub fn make_key_pair(name: &'static str, u: u32) -> (T::AccountId, Option<SecretKey>) {
```

Comment:
```suggestion
    fn make_key_pair(name: &'static str, u: u32) -> (T::AccountId, Option<SecretKey>) {
```
`UserBuilder` is a chainable builder pattern, so it should have only one method that return non-Self object, its `build` method.
I've made another suggestion in the file you are using this private fn.